source/win-preferences/schema/editor.ts

```diff
@@ -282,6 +282,11 @@ export function getEditorFields (config: ConfigOptions): PreferencesFieldset[] {
           label: trans('Show formatting toolbar when text is selected'),
           model: 'editor.showFormattingToolbar'
         },
+        {
+          type: 'checkbox',
+          label: trans('Show Line Numbers'),
+          model: 'editor.showLineNumbers'
```

Comment:
```suggestion
          label: trans('Show Line Numbers for Markdown files'),
          model: 'editor.showMarkdownLineNumbers'
```

Same as above + wording